<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="4" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-12T13:36:16 IST" name="Master Suite" finished-at="2024-09-12T13:37:32 IST" duration-ms="75685">
    <groups>
      <group name="Master">
        <method signature="TC001_AccountRegistrationTest.verify_account_Registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@397fbdb]" name="verify_account_Registration" class="testCases.TC001_AccountRegistrationTest"/>
        <method signature="TC002_Login_Test.veryfy_login()[pri:0, instance:testCases.TC002_Login_Test@2d3379b4]" name="veryfy_login" class="testCases.TC002_Login_Test"/>
      </group> <!-- Master -->
      <group name="Data driven">
        <method signature="TC003_Login_DDT.veryfy_login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_Login_DDT@30c15d8b]" name="veryfy_login" class="testCases.TC003_Login_DDT"/>
      </group> <!-- Data driven -->
      <group name="Sanity">
        <method signature="TC002_Login_Test.veryfy_login()[pri:0, instance:testCases.TC002_Login_Test@2d3379b4]" name="veryfy_login" class="testCases.TC002_Login_Test"/>
      </group> <!-- Sanity -->
      <group name="Regression">
        <method signature="TC001_AccountRegistrationTest.verify_account_Registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@397fbdb]" name="verify_account_Registration" class="testCases.TC001_AccountRegistrationTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2024-09-12T13:36:16 IST" name="Test" finished-at="2024-09-12T13:37:32 IST" duration-ms="75685">
      <class name="testCases.TC003_Login_DDT">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_Login_DDT@30c15d8b]" started-at="2024-09-12T13:36:54 IST" name="setup" finished-at="2024-09-12T13:37:01 IST" duration-ms="6296" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="veryfy_login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_Login_DDT@30c15d8b]" started-at="2024-09-12T13:37:03 IST" name="veryfy_login" data-provider="Logindata" finished-at="2024-09-12T13:37:07 IST" duration-ms="3822" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[pavanoltraining@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veryfy_login -->
        <test-method signature="veryfy_login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_Login_DDT@30c15d8b]" started-at="2024-09-12T13:37:07 IST" name="veryfy_login" data-provider="Logindata" finished-at="2024-09-12T13:37:11 IST" duration-ms="4305" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[lakshmi@yahoo.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Laxmi]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at testCases.TC003_Login_DDT.veryfy_login(TC003_Login_DDT.java:65)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veryfy_login -->
        <test-method signature="veryfy_login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_Login_DDT@30c15d8b]" started-at="2024-09-12T13:37:12 IST" name="veryfy_login" data-provider="Logindata" finished-at="2024-09-12T13:37:14 IST" duration-ms="2690" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[laksh@yahoo.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Lakshmi]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at testCases.TC003_Login_DDT.veryfy_login(TC003_Login_DDT.java:65)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veryfy_login -->
        <test-method signature="veryfy_login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_Login_DDT@30c15d8b]" started-at="2024-09-12T13:37:15 IST" name="veryfy_login" data-provider="Logindata" finished-at="2024-09-12T13:37:26 IST" duration-ms="11604" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[laks@yahoo.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[xyz]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veryfy_login -->
        <test-method signature="veryfy_login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_Login_DDT@30c15d8b]" started-at="2024-09-12T13:37:26 IST" name="veryfy_login" data-provider="Logindata" finished-at="2024-09-12T13:37:31 IST" duration-ms="4608" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[abc123@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veryfy_login -->
        <test-method is-config="true" signature="closebrowser()[pri:0, instance:testCases.TC003_Login_DDT@30c15d8b]" started-at="2024-09-12T13:37:31 IST" name="closebrowser" finished-at="2024-09-12T13:37:32 IST" duration-ms="513" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closebrowser -->
      </class> <!-- testCases.TC003_Login_DDT -->
      <class name="testCases.TC001_AccountRegistrationTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_AccountRegistrationTest@397fbdb]" started-at="2024-09-12T13:36:16 IST" name="setup" finished-at="2024-09-12T13:36:23 IST" duration-ms="6555" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verify_account_Registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@397fbdb]" started-at="2024-09-12T13:36:23 IST" name="verify_account_Registration" finished-at="2024-09-12T13:36:25 IST" duration-ms="2456" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_account_Registration -->
        <test-method is-config="true" signature="closebrowser()[pri:0, instance:testCases.TC001_AccountRegistrationTest@397fbdb]" started-at="2024-09-12T13:36:25 IST" name="closebrowser" finished-at="2024-09-12T13:36:26 IST" duration-ms="357" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closebrowser -->
      </class> <!-- testCases.TC001_AccountRegistrationTest -->
      <class name="testCases.TC002_Login_Test">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC002_Login_Test@2d3379b4]" started-at="2024-09-12T13:36:26 IST" name="setup" finished-at="2024-09-12T13:36:31 IST" duration-ms="5368" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="veryfy_login()[pri:0, instance:testCases.TC002_Login_Test@2d3379b4]" started-at="2024-09-12T13:36:31 IST" name="veryfy_login" finished-at="2024-09-12T13:36:53 IST" duration-ms="21832" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testCases.TC002_Login_Test.veryfy_login(TC002_Login_Test.java:48)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veryfy_login -->
        <test-method is-config="true" signature="closebrowser()[pri:0, instance:testCases.TC002_Login_Test@2d3379b4]" started-at="2024-09-12T13:36:53 IST" name="closebrowser" finished-at="2024-09-12T13:36:54 IST" duration-ms="1115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closebrowser -->
      </class> <!-- testCases.TC002_Login_Test -->
    </test> <!-- Test -->
  </suite> <!-- Master Suite -->
</testng-results>
